event hubs migration guide.md

```diff
@@ -0,0 +1,118 @@
+# Migration Guide (EventHubs v2 to v5)
+
+## Class changes
+
+`EventHubClient` has been split into two clients - `EventHubProducerClient` (for sending
+messages) and `EventHubConsumerClient` (for receiving messages).
+
+`EventProcessorHost`'s functionality has been merged into `EventHubConsumerClient`.
+
+| In v2                                          | Equivalent in v5                                                 | Sample |
```

Comment:
This may be more helpful and align better with samples if it were framed in scenarios rather than focused on a client.  Especially in cases where there may be multiple samples that would be relevant and helpful.

Reply:
I broke them apart a bit just to give them a little separation - so I have receiving, sending and construction as three separate little tables instead of one single table.